### PR TITLE
Scaffolding for route configurations

### PR DESCRIPTION
**Pull request by hannahhenderson**
Allows front-end to ping routes and get a test response

### CODE REVIEW

File: server/api/expense-pool/expense-pool.controller.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  // otherwise send data
+  res.send('getPool');
+};
+
+// updatePool
+controller.updatePool = (req, res) => {
+  res.send('updatePool');
+};
+
+// module.exports = {
```
Comment (dting) [inline server/api/expense-pool/expense-pool.controller.js]: Let's get rid of this block of code.

File: server/api/expense-pool/expense/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import controller from './expense.controller';
+// import itemRouter from express.Router({ mergeParams: true });
```
Comment (dting) [inline server/api/expense-pool/expense/index.js]: We can remove this comment and actually just use it on the line below.

```
const router = express.Router({ mergeParams: true });
```

File: server/api/expense-pool/expense-pool.controller.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+const controller = {};
+
+// addPool
```
Comment (dting) [inline server/api/expense-pool/expense-pool.controller.js]: This just restates the function name. We can remove this comment.

Same for the other inline function comments below. 

We might want to create block comments for these functions at a later time to indicate what we expect to be on the `req`. For example, what `req.body` and maybe if authentication is required.

**Comment by dting** [comment]
Couple of notes about the comments. Other than that LGTM. Let me know if you want me to take another look or go ahead and merge it up after removing some of the latent comment blocks.

**Comment by hannahhenderson** [comment]
Changes incorporated

**Pull request merged**